besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1185,11 +1185,13 @@ public void run() {
       // Need to create vertx after cmdline has been parsed, such that metricsSystem is configurable
       vertx = createVertx(createVertxOptions(metricsSystem.get()));
 
-      final BesuCommand controller = validateOptions().configure().controller();
+      validateOptions()
+          .configure()
+          .initController()
+          .startPlugins()
+          .preSynchronization()
+          .startSynchronization();
```

Comment:
I appreciate the consistency, but at a glance I don't see a reason we need the builder pattern at all. Can this just become:
```
  validateOptions();
  configure();
  initController();
  startPlugins();
  preSynchronization();
  startSynchronization();
```